test/integration/targets/ec2_elb/tasks/main.yml

```diff
@@ -0,0 +1,536 @@
+---
+- block:
```

Comment:
I think `block` has a `environment:` attribute that you can use to set all the aws secret/key id/session token variables as their bash appropriate versions (AWS_SECRET_KEY etc) and not have to do so on every module call. 

For the tests checking that the module fails correctly when missing credentials, put them in a separate block.